src/ui/item_list_view.c

```diff
@@ -471,13 +471,19 @@ item_list_view_update_item_internal (ItemListView *ilv, itemPtr item, GtkTreeIte
 	gchar		*title, *time_str;
 	const GIcon	*state_icon;
 	gint		state = 0;
+        gboolean        fixeddatefmt;
```

Comment:
Regarding indentation: you do not need to fix it... But let me ask which IDE do you use? Would having VIM modelines in the headers help for auto indentation?

Reply:
@lwindolf I use vim as my ide.  I'm not sure that I see the indent issue you've identified in this snippet. 
I suspect it might have something to do with expandtab being turned on.

Reply:
@lwindolf it is expandtab, I've fixed the spacing issue.